scripts/New-VersionConstantsFile.ps1

```diff
@@ -18,23 +19,21 @@ $versionConstantsFileContents =
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.Reflection;
-
 namespace $namespace
 {
     public static class VersionConstants
     {
-        public const string PreRelease = "$versionSuffix";
+        public const string Prerelease = "$versionSuffix";
         public const string AssemblyVersion = "$versionPrefix";
         public const string FileVersion = AssemblyVersion + ".0";
-        public const string Version = AssemblyVersion + PreRelease;
+        public const string Version = AssemblyVersion + Prerelease;
     }
 }
 "@
 
-if (-not (Test-Path $outputDirectory)) {
-    New-Item -ItemType Directory -Path $outputDirectory | Out-Null
-}
+$outputFile = "$SourceRoot\Sarif\VersionConstants.cs"
```

Comment:
We generate the file in the source directory now, so there's no need to check and create the directory first.